2by2.py

```diff
@@ -34,7 +34,25 @@ def turn_u(self):
         self.faces['u'][0][1], self.faces['u'][1][1], self.faces['u'][0][0], self.faces['u'][1][0] = self.faces['u'][0][0], self.faces['u'][0][1], self.faces['u'][1][0], self.faces['u'][1][1]
         # Changing other sides
         self.faces['l'][0][0], self.faces['l'][0][1], self.faces['b'][0][0], self.faces['b'][0][1], self.faces['f'][0][0], self.faces['f'][0][1], self.faces['r'][0][0], self.faces['r'][0][1] = self.faces['f'][0][0], self.faces['f'][0][1], self.faces['l'][0][0], self.faces['l'][0][1], self.faces['r'][0][0], self.faces['r'][0][1], self.faces['b'][0][0], self.faces['b'][0][1]
-
+    def turn_l(self):
```

Comment:
Is there a way to generalize these rotations into your own functions so this approaches legibility? You could pass in the starting face and target face so you don't have to self.faces a million times at least. Consider a way where you don't need a giant ugly 1-line assignment. 

- Hal